arch/s390/dfltcc_detail.h

```diff
@@ -162,6 +162,27 @@ typedef enum {
 #define HB_BITS 15
 #define HB_SIZE (1 << HB_BITS)
 
+static inline void get_history_lengths(struct dfltcc_param_v0 *param, size_t *hl_high, size_t *hl_low) {
```

Comment:
**nitpick:** 
These three functions are not in such a central part of the code, and are somewhat easier to figure out, but I'd still appreciate if they got a one-liner comment each.